### PR TITLE
Math magicians: fetch data from API

### PR DESCRIPTION
**Pull request by techmoves**
in this repo i did this :

- [x] Use the following external API: [API Ninjas](https://api-ninjas.com/api/quotes)

    

- [x] Created a new component for displaying a quote

   

- [x]  Fetch data inside of a new component

 

- [x]    Show a loading state

   

- [x]  Show an error state

   

- [x]  Imported the new component into your main component (next to the Calculator)

### CODE REVIEW

File: src/componets/qoutes.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+/*eslint-disable*/
```
Comment (DuaneDave) [inline src/componets/qoutes.js]: Consider enabling the linter to perform checks on this file in order for it to pick up possible syntax errors in your code.

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @techmoves 👋 , 

Good job so far on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ The data is being fetched with multiple states 👍 
✔️ Correct usage of use effect

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tugsem** [comment]
## Invalid Code Review Request ❌ 

Hi @techmoves , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
image

<h1>You have submitted a project, that fails in linters</h2>


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by codecaiine — APPROVED** [review]
Hi @techmoves , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good implementation of  the requested changes ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**